priv/repo/seeds.exs

```diff
@@ -1,11 +1,2 @@
-# Script for populating the database. You can run it as:
-#
-#     mix run priv/repo/seeds.exs
-#
-# Inside the script, you can read and write to any of your
-# repositories directly:
-#
-#     App.Repo.insert!(%App.SomeSchema{})
-#
-# We recommend using the bang functions (`insert!`, `update!`
-# and so on) as they will fail if something goes wrong.
+App.Ctx.create_status(%{text: "verified"})
+App.Ctx.create_status(%{text: "unverified"})
```

Comment:
@SimonLab unverified should be `0` therefore we might need to insert it first ... 💭